patches/gcc/linaro-6.3-2017.02/952-bionic-errno.patch

```diff
@@ -0,0 +1,19 @@
+commit 6cd4ad106ef87a3c58b0c3478e78409b47000de0
```

Comment:
I see that you're only patching the linaro-6.3 version. Does it build with stock 6.3, without these patches?